.github/issue_template.md

```diff
@@ -0,0 +1,19 @@
+---
+name: openHABian issue template
+about: A template issue for the openHABian repository.
+labels: new
+---
+## Issue information:
+Please be as detailed as possible to help us help you. Also, please be sure to
+check that your issue has not already been reported to the [openHAB community](https://community.openhab.org/t/13379)
+or to this GitHub issue tracker.
```

Comment:
give pointer to remove "is: open" from Github standard search as that'll only give open issues in  return so similar or matching closed issues don't show up.
Also prominently add that hint to debug guide.

Reply:
ok